client/src/index.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import '../style.sass';
```

Comment:
What does ../style do vs ./style?

Reply:
`./foo` means file `foo` in the same directory, where `../bar` means file `bar` in the parent directory, its just going back one level